src/components/Banner.jsx

```diff
@@ -7,76 +7,75 @@ class Banner extends React.Component {
 
   renderLogos = allLogos => (
     <Card fluid >
-      <Card.Content className="card">
-        {this.getRandom(allLogos, 4).map(logo => {
-        if(logo.node.image.file.url != null){
+      <Card.Content className="cardcontent">
+        {getRandom(allLogos, 5).map(logo => {
           return(
             <Image className="image" key={logo.node.id} size="small" src={logo.node.image.file.url} />
           );
-        }
-        return null;
-      })}
+        })}
       </Card.Content>
     </Card>
   )
 
-  getRandom = (arr, n) => {
-    var result = new Array(n),
-    len = arr.length,
-    taken = new Array(len);
-
-    if(n > len)
-      return arr;
-
-    while(n--){
-      var x = Math.floor(Math.random() * len);
-      result[n] = arr[x in taken ? taken[x] : x];
-      taken[x] = --len in taken ? taken[len] : len;
-    }
-    return result;
-  }
-
   render(){
     return(
-      <BannerWrapper>
+      <BannerWrapper className="banner">
         {this.renderLogos(this.props.data.allContentfulBannerLogo.edges)}
         </BannerWrapper>
       );
   }
 }
 
+const getRandom = (arr, n) => {
```

Comment:
I see you move the function outside of the component class, but it still doesn't have much to with the component, so i think it also shouldn't be in this file and after some googling most people agree that it should have it's separate file. So probably best to move it to something like src/utils.js

Reply:
sure :)